pkg/vtpm/src/server.cpp

```diff
@@ -72,7 +145,6 @@ int readMessage(int sock, google::protobuf::uint32 size)
     ostringstream command;
     command << request.command() << " " << "> cmd.output 2>&1";
     system(command.str().c_str());
```

Comment:
exec would make more sense here, but there might be other sanitization needed.

Reply:
moved to execve()